### ISSUE TITLE
Fix ruby task of travis (internal link checker for release-downloads)

### ISSUE THREAD

**Issue by jo-pol** [d-bl/GroundForge#116]
The ruby task fails for the first time with build [679.3](https://travis-ci.org/d-bl/GroundForge/builds/363715344), it succeeds for the last time with build [#678.3](https://travis-ci.org/d-bl/GroundForge/builds/362267275) but keeps failing when restarting [730.3](https://travis-ci.org/d-bl/GroundForge/builds/423368358)

```
$ htmlproofer ./_site --disable-external --empty_alt_ignore  --url-ignore "#"
htmlproofer 3.8.0 | Error:  Whoops, we can't understand your command.
htmlproofer 3.8.0 | Error:  invalid option: --empty_alt_ignore
htmlproofer 3.8.0 | Error:  Run your command again with the --help switch to see available options.
```

**Comment by jo-pol** [comment]
At last: the problem was `_`'s that should be `-`'s.
Now the link checker does its work again, the broken link should be fixed.

**Comment by jo-pol** [comment]
fixed with b90ff60
